src/main/java/org/checkerframework/checker/builder/TypesafeBuilderAnnotatedTypeFactory.java

```diff
@@ -465,6 +477,27 @@ private static boolean hasAnnotation(
         .anyMatch(anm -> AnnotationUtils.areSameByClass(anm, annotClass));
   }
 
+  private static String autoValuePropToBuilderSetterName(
+      String prop, Set<String> allBuilderMethodNames) {
+    // property name may be prefixed JavaBean style with 'get' or 'is'; strip that part if present
+    if (prop.startsWith("get") && Character.isUpperCase(prop.charAt(3))) {
```

Comment:
Is it possible for the entire name of the method to be `get()`? For example, what if I named the property "get". Would it be possible for me to direct AutoValue to produce a method that doesn't have a prefix and then cause this code to throw a `StringOutOfBoundsException`

Reply:
Good catch!  I added a fix and a test